.github/workflows/update-docs-in-website.yml

```diff
@@ -37,10 +37,10 @@ jobs:
         run: |
           rm -r ./markdown/docs/tools/generator
           mkdir -p ./markdown/docs/tools/generator
-          rm ../generator/docs/README.md
-          rm -r ../generator/docs/jsdoc2md-handlebars
+          rm ./apps/generator/docs/README.md
```

Comment:
actually I think it should be `../generator/apps/generator/docs/README.md`

it is executed in context of `working-directory: ./website`, so needs to go out and into `generator` first (generator as repo not as app)